src/applications/caregivers/components/SubmitError/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+const SubmitError = () => {
+  const ErrorBody = () => {
+    return (
+      <div>
+        <p>
+          We’re sorry. Something went wrong when you tried to submit your
+          application. Please review your application to make sure you entered
+          your information correctly. If you think your information is correct,
+          please call the VA.gov help desk at 855-574-7286 (TTY: 711). We’re
```

Comment:
Should the phone number & TTY number both be in a `<Telephone />` component?

Reply:
Went ahead and :rip:ed all the phone numbers for the `<Telephone />` component